addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -10534,7 +10534,7 @@ QUnit.module("Views", (hooks) => {
 
         // since the field image is not set, kanban_image will generate an URL
         const imageOnRecord = target.querySelectorAll(
-            'img[data-src*="/web/image"][data-src*="&id=1"]'
+            'img[data-src*="/web/image/partner/1/image"]'
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)